apps/portals/src/Navbar.tsx

```diff
@@ -214,7 +214,7 @@ function Navbar() {
                     setShowLoginDialog(true)
                   }}
                 >
-                  SIGN&nbsp;IN
+                  Sign&nbsp;In
```

Comment:
Text transform only changes the [first letter of each word](https://stackoverflow.com/a/52166626), so update content